.github/workflows/rust.yml

```diff
@@ -240,7 +240,7 @@ jobs:
       - name: Cache build artifacts
         uses: Swatinem/rust-cache@v2
       - name: Install dependencies
-        run: dnf install -y gcc git-core libzstd-devel openssl-devel util-linux awk
+        run: dnf install -y gcc git-core libzstd-devel openssl-devel util-linux util-linux-script awk
```

Comment:
is util-linux-script an RPM? my system seems to not think so.

Reply:
Odd, it is on mine. I tired to run with util-linux rpm only and got that the "script" command was not found. I then Locally tried to run a script and got the suggestion to install util-linux-script and then locally that worked, so I added it to the ci, (which it seems to have fixed the docs)

Reply:
ok. it was broken out to a subpackage in F42, which explains why I didn't see it (i'm on F41).

👍